### PR TITLE
Refactor init of DRS preference to fix crash on first attempt to select it.

### PR DESCRIPTION
**Pull request by abioteau**
This commit solve issue https://github.com/sonyxperiadev/bug_tracker/issues/131

### CODE REVIEW

File: src/sonyxperiadev/extendedsettings/ExtendedSettingsActivity.java [hunk 1]
```diff
@@ -174,7 +167,7 @@ public boolean isOnlyRefreshRateChange(Object obj) {
         }
     }
 
-    private static LinkedList<DisplayParameters> thisDp = new LinkedList<>();
+    private static LinkedList<DisplayParameters> mDp;
```
Comment (luk1337) [inline src/sonyxperiadev/extendedsettings/ExtendedSettingsActivity.java]: sDp.

File: src/sonyxperiadev/extendedsettings/ExtendedSettingsActivity.java [hunk 2]
```diff
@@ -263,6 +256,12 @@ protected void onCreate(Bundle savedInstanceState) {
         mPrefEditor.apply();
     }
 
+    @Override
+    protected void onResume() {
+        super.onResume();
+
```
Comment (luk1337) [inline src/sonyxperiadev/extendedsettings/ExtendedSettingsActivity.java]: nuke that empty line.

File: src/sonyxperiadev/extendedsettings/ExtendedSettingsActivity.java (hunk 1 above)
Comment (abioteau) [reply src/sonyxperiadev/extendedsettings/ExtendedSettingsActivity.java]: Done

File: src/sonyxperiadev/extendedsettings/ExtendedSettingsActivity.java (hunk 2 above)
Comment (abioteau) [reply src/sonyxperiadev/extendedsettings/ExtendedSettingsActivity.java]: Removed

**Pull request merged**